extensions/wikia/EmbeddableDiscussions/styles/SettingsModal.scss

```diff
@@ -0,0 +1,106 @@
+@import 'skins/shared/color';
+@import 'skins/shared/mixins/flexbox';
+
+.embeddable-discussions-settings-modal.modal {
```

Comment:
Class names are really long, is this a convention in app extensions? In my opinion they make code more difficult to read.